includes/Model/Story.php

```diff
@@ -104,9 +133,10 @@ public function load_from_post( $_post ) {
 			return false;
 		}
 
-		$this->title  = get_the_title( $post );
-		$this->markup = $post->post_content;
-		$this->url    = (string) get_permalink( $post );
+		$this->title   = $post->post_title;
```

Comment:
Why was this changed from using `get_the_title`?

Reply:
See explanation at https://github.com/rtCamp/web-stories-wp/pull/323#discussion_r581761395

Admittedly, we could still use `get_the_titlle()`. But `get_the_excerpt()` would cause an infinite loop.

Reply:
I understand `get_the_excerpt`, `get_the_title` I don't understand.